open-machine-learning-jupyter-book/assignments/set-up-env/first-assignment.ipynb

```diff
@@ -24,22 +24,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
Why your submit this file to the PR? This does not suppose to be your change in this PR.

Reply:
Sorry, this is the homework that was “ramped up“ before, I didn't have PR before, and I deleted him now